.gitignore

```diff
@@ -1,2 +1,4 @@
 .coverage
 .tox/
+*.pyc
+*.egg-info/
```

Comment:
you mean the  `egg-info/` folder? Probably not, I just followed the same done here:
https://github.com/Stanford-Online/xblock-free-text-response/blob/master/.gitignore
but I can remove it if necessary.